action.yml

```diff
@@ -11,13 +11,18 @@ inputs:
     default: 8585
   requested_network:
     description: "Requested network to connect to"
+    required: true
+    default: 1
+  wallet_connect_project_id:
+    description: "Your WalletConnect project id [defaults to secret WALLET_CONNECT_PROJECT_ID]"
+    required: true
```

Comment:
qq - do we need `required: true` if we set default here? just one less thing need to update / add to current seacrest implementations if not

Reply:
ah nvm, i think this is referring to the step. maybe just remove duplicate line then